packages/react/src/lib/felaRenderer.tsx

```diff
@@ -7,6 +7,13 @@ import rtl from 'fela-plugin-rtl'
 
 import { Renderer } from '../themes/types'
 
+// Blacklist contains a list of classNames that are used by FontAwesome
+// https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use
+const blacklistedClassNames = ['fa', 'fas', 'far', 'fal', 'fab']
```

Comment:
a bit concerned about these as they can change at any time in fontawesome and we have no control over that..

Reply:
agree - especially given that each theme might load CSS libraries with different set of class names that should be blacklisted. Would expect that this `blacklisted` information will be defined by theme, not (only) general Stardust logic - and, thus, would be extendable